src/app/services/itinerary.service.ts

```diff
@@ -10,12 +11,14 @@ export class ItineraryService {
 
   constructor(private httpClient: HttpClient) { }
 
-  hostURL = 'http://localhost:3000/';
+  hostURL = 'https://be-dev-yourney.herokuapp.com/';
```

Comment:
Codacy has a fix for the issue: [Strings must use doublequote.](https://app.codacy.com/gh/ispp-yourney/angular-app/pullRequest?prid=7157930)
```suggestion
  hostURL = "https://be-dev-yourney.herokuapp.com/";
```